### PR TITLE
Readd styleguide.scss linter fixes after they got lost in a merge

### PR DESCRIPTION
**Pull request by nickjs**
@rogeruiz 
Seems like all those fixes got lost in the merge, so here they are again :)

### CODE REVIEW

**Comment by nickjs** [comment]
Interesting, so the reason CI fails is because we switched to using the Ruby gem for the styleguide, but the styleguide doesn't have the newest copy of the assets from here yet, which has the new `$color-black`.

**Comment by rogeruiz** [comment]
Thanks @nickjs for opening this back up. Right now the `scss-lint` is going to be a part of #980. Let's hold on merging, but we can keep it open, this in at the moment. Once `18f-pages-staging` is more stable we can review the code against this PR. I'll make sure these changes aren't lost in the shuffle again. Thanks!

**Comment by rogeruiz** [comment]
Hey @nickjs. Good news! #980 is merged in. We'll need to rebase the branch onto `18f-pages-staging` and resolve any code conflicts. Once that's done, I can review this and merge it in. Thanks!

**Comment by nickjs** [comment]
@rogeruiz Rebased, should be good to go, sorry for the delay

**Comment by rogeruiz** [comment]
Awesome. Thanks @nickjs. once this lands in `18f-pages-staging`, can you rebase #987 as well? I'll merge this in once it passes. :+1: :rocket:

**Comment by nickjs** [comment]
See above comment for why this still fails on CI. Basically the gem hasn't been updated with what's in `18f-pages-staging` but `styleguide.scss` imports the gem version of the styleguide instead of the version in `src/`

**Comment by rogeruiz** [comment]
@nickjs ah good catch. We've been discussing this here 18F/us_web_design_standards_gem#15 about whether or not the gem should include the source files or if it should include the compiled files. Seems like we should refactor our gem process to include the bundled / compiled JS and keep the source Sass files within the gem.

**Comment by maya** [comment]
The fixes look good to me.

File: docs/assets/css/styleguide.scss [hunk 1]
```diff
@@ -279,13 +264,19 @@ $site-top:           124px;
 
 .preview {
   @include clearfix();
-  background-color: #fff;
   @include margin(2em null);
+  background-color: $color-white;
   padding: 1em $site-margins;
-  border: 1px solid #eeeeee;
+  border: 1px solid $color-gray-warm-light;
```
Comment (maya) [inline docs/assets/css/styleguide.scss]: `#eeeeee` is not the same as `$color-gray-warm-light`, so let's switch back.

**Comment by maya** [comment]
@nickjs hey there, it looks like this branch has merge conflicts that must be resolved. Let us know if you can resolve these. Thanks!

**Comment by msecret** [comment]
I'll look into fixing the merge conflicts with this

**Comment by maya** [comment]
Hey @nickjs, will you be able to integrate the merge conflict fixes in https://github.com/18F/web-design-standards/pull/1063? We'd love to get this in.

**Comment by rogeruiz** [comment]
Closing this PR in favor of #1180 which will `cherry-pick` these commits.

**Pull request closed without merging**